src/index.js

```diff
@@ -523,7 +523,7 @@ function createConfig(options, entry, format, writeMeta) {
 					useTypescript &&
 						typescript({
 							typescript: require('typescript'),
-							cacheRoot: `./.rts2_cache_${format}`,
+							cacheRoot: `./node_modules/.rts2_cache_${format}`,
```

Comment:
clever.

Reply:
Rekon we could take that 1 step further with,

```suggestion
							cacheRoot: `./node_modules/.rts2_cache_${format}`,
							cacheRoot: `./node_modules/.cache/.rts2_cache_${format}`,
```

Reply:
was kinda annoyed with those 3 directories in my project dir 🙃